src/components/SiteHeader/index.js

```diff
@@ -206,6 +61,7 @@ const MenuItems = ({ onLogOut, showItems, onLinkClick }) => {
           onClick={onLinkClick}
           forwardedAs="a"
```

Comment:
Helaas, de link Meldingen blijft niet rood.

Reply:
I don't have a "meldingen" option...

<img width="1494" alt="Screenshot 2023-03-27 at 11 33 52" src="https://user-images.githubusercontent.com/46756331/227902842-a9f2c813-db42-464d-a942-26d222c31769.png">

Reply:
that's strange. It does work with me...